gridcomps/ExtData2G/CMakeLists.txt

```
@@ -0,0 +1,32 @@
+esma_set_this (OVERRIDE MAPL.ExtData2G)
+
+set (srcs
+     ExtDataFileStream.F90
+     ExtDataRule.F90
+     ExtDataDerived.F90
+     ExtDataConfig.F90
+     ExtDataGridCompNG.F90
+     TimeStringConversion.F90
+     ExtDataTypeDef.F90
+     ExtDataOldTypesCreator.F90
+     ExtDataBracket.F90
+     ExtDataUpdatePointer.F90
+     ExtDataAbstractFileHandler.F90
+     ExtDataClimFileHandler.F90
+     ExtDataSimpleFileHandler.F90
+     ExtDataNode.F90
+     ExtDataLgr.F90
+     ExtDataConstants.F90
+     ExtDataSample.F90
+     ExtData_IOBundleMod.F90
+     ExtData_IOBundleVectorMod.F90
+    )
+
+
+esma_add_library (${this} SRCS ${srcs} DEPENDENCIES MAPL.shared MAPL.base MAPL.generic MAPL.griddedio TYPE SHARED)
+target_link_libraries (${this} PUBLIC GFTL::gftl GFTL_SHARED::gftl-shared esmf NetCDF::NetCDF_Fortran
```

```suggestion
target_link_libraries (${this} PUBLIC GFTL::gftl GFTL_SHARED::gftl-shared YAFYAML::yafyaml esmf NetCDF::NetCDF_Fortran
```

I think this will fix the UFS build test.